### ISSUE TITLE
Attempt to query a type of just ceasn:Competency results in a Gateway timeout.

### ISSUE THREAD

**Issue by mparsons-ce** [CredentialEngine/CredentialRegistry#335]
Using Postman, I submitted a query for just Competencies (in order to get the total number)
{
	"@type": "ceasn:Competency"
}
I get a timeout error:
![image](https://user-images.githubusercontent.com/58825866/83771363-d08cd200-a647-11ea-826d-fa8e859b734b.png)

If I submit a query for frameworks (ceasn:CompetencyFramework), the query works - same for other top level classes.

**Comment by excelsior** [comment]
This is the problem Nate discovered last year: If a large traversal is used inside a `union()` step, it behaves poor performance-wise. See the `Search timeout when using negation` email chain from  August 15, 2019 for more details.)

I've tried a few things in the hope of making the query perform better, but to no avail. I ended up increasing the timeouts to one minute, now the query succeeds for me.
